src/mongocrypt-crypto.c

```diff
@@ -147,12 +151,15 @@ static bool
 _crypto_aes_256_ctr_encrypt (_mongocrypt_crypto_t *crypto, aes_256_args_t args)
 {
    mongocrypt_status_t *status = args.status;
-   if (args.key->len != MONGOCRYPT_ENC_KEY_LEN) {
+
+   BSON_ASSERT_PARAM (crypto);
+
+   if (!args.key || args.key->len != MONGOCRYPT_ENC_KEY_LEN) {
       CLIENT_ERR ("invalid encryption key length");
       return false;
    }
 
-   if (args.iv->len != MONGOCRYPT_IV_LEN) {
+   if (!args.iv || args.iv->len != MONGOCRYPT_IV_LEN) {
```

Comment:
Suggest asserting `args.iv` as not-null instead.

Reply:
:+1: Change coming in next commit.